dotcom-rendering/.eslintrc.js

```diff
@@ -97,4 +97,14 @@ module.exports = {
 			'babel-module': { extensions: ['.ts', '.tsx', '.js'] },
 		},
 	},
+	overrides: [
+		{
+			files: ['**/**.js'],
+			rules: {
+				'@typescript-eslint/no-var-requires': 'off',
+				'@typescript-eslint/no-unsafe-member-access': 'off',
+				'@typescript-eslint/no-misused-promises': 'off',
+			},
+		},
+	],
```

Comment:
I added these because it was getting too annoying to work on vanilla JS files. I have no idea why typescript-eslint was trying to apply these rules in the first place